packages/ruleset/src/functions/array-boundary.js

```diff
@@ -0,0 +1,75 @@
+const { validateSubschemas } = require('../utils');
+
+module.exports = function(schema, _opts, { path }) {
+  return validateSubschemas(schema, path, arrayBoundaryErrors, true, false);
+};
+
+const debugEnabled = false;
+function debug(msg) {
+  if (debugEnabled) {
+    console.log(msg);
+  }
+}
+
+const bypassFormats = {
+  minItems: [],
+  maxItems: []
+};
+
+function arrayBoundaryErrors(schema, path) {
+  const errors = [];
+
+  if (schema.type === 'array') {
+    if (
+      isUndefinedOrNull(schema.minItems) &&
+      !bypassFormats.minItems.includes(schema.format)
+    ) {
+      errors.push({
+        message: 'Should define a minItems for a valid array',
+        path
+      });
+      debug('>>> minItems field missing for: ' + path.join('.'));
+    }
+    if (
+      isUndefinedOrNull(schema.maxItems) &&
+      !bypassFormats.maxItems.includes(schema.format)
+    ) {
+      errors.push({
+        message: 'Should define a maxItems for a valid array',
+        path
+      });
+      debug('>>> maxItems field missing for: ' + path.join('.'));
+    }
+    if (
+      !isUndefinedOrNull(schema.minItems) &&
+      !isUndefinedOrNull(schema.maxItems) &&
+      schema.minItems > schema.maxItems
+    ) {
+      errors.push({
+        message: 'minItems cannot be greater than maxItems',
```

Comment:
This duplicates [the logic in `property-attributes`](https://github.com/IBM/openapi-validator/blob/main/packages/ruleset/src/functions/property-attributes.js#L56)

Reply:
ok，I will delete this part.